### DISCUSSION TITLE
uWidgets 0.6.0

### DISCUSSION THREAD

**Discussion by creewick** [creewick/uWidgets#45] [Announcements]
First of all, it's a **performance update.**
We fixed some problems that were leading to high memory usage.

Also, there are some changes in themes. Now we offer more options to choose from. 
Maybe we'll allow to save custom themes to that list in the future.

<img width="1018" alt="image" src="https://github.com/user-attachments/assets/026cf5b3-7927-4b97-91f6-4cecb1d8f162">

I see a lot of you sent some suggestions and bug requests. Thank you for your interest, I will look through each of them. 
See you in the future releases.

**Comment by Lap84** [comment]
Hello. Yesterday I've installed uwidgets and I'm loving it. My desktop is with a new and clean face now.
Thanks a lot for your work.
Is there any chance to implement Portuguese (🇵🇹) language. I can help with the translation if you need.
Sorry if this isn't the right place to ask for this, but didn't found other one.
Have a great year.
